src/_blog/SRI-1.0.0.md

```diff
@@ -0,0 +1,73 @@
+
+![Config A](/assets/config-a.svg)
+
+The newly implemented **pool fallback functionality** ensures that if a pool rejects a miner's template, the miner's Job Declarator Client automatically switches to an alternative pool. Miners can configure several fallback options and if all listed pools decide to censor, miners will end up solo mining. This serves as a powerful incentive for pools to act in the best interest of miners, as rejecting templates without a valid reason could lead to a significant loss of hash power to competitors. If all miners run standardized software, the entire hash rate of the pool could end up with a competing pool.
+
+The rejection of templates triggering fallback could be happen in several other scenarios, asides from the pool rejecting the template in order to censor transactions:
+
+- A bug in the pool or downstream
+- A connection interruption between the pool and downstream
+- Downstream creates custom templates that are below acceptable fee policy defined by the pool
+
+**SRI’s pool** is simple and opinionated and currently can receive a template from the Template Provider, and create jobs for the downstream with the adequate difficulty. Anyone can easily deploy this simple pool today or use our libraries to integrate into their existing operations. Going forward, we plan to continue refactoring our code to allow for more diverse use-cases. Anyone interested in integrating SV2 into an existing pool, or starting a brand-new pool, [let us know](https://discord.gg/stEjAZ2y8r), we’d love to collaborate more closely. One example of a pool using SRI in production is [DMND](https://dmnd.work).
+
+**Template provider** enables the selection of transactions from the locally-run bitcoin node. Miners (soon, also independent third parties) can create custom block templates and declare custom mining jobs to the pool. Currently, to use the Template Provider you need to use our patched version of the Bitcoin Core. Sjors Provoost, opened a pull request in the Bitcoin Core [#29432](https://github.com/bitcoin/bitcoin/pull/29432) which can be tested and reviewed. We’re inviting code reviewers to take a look at smaller PR’s that build [#2943](https://github.com/bitcoin/bitcoin/pull/29432). 
+
+## 🙏 Thank you!
+
+SRI 1.0.0, wouldn’t be possible without our contributors, supporters, working group, testers, and community.  Since our last update, our contributor base has grown even further!  What once started as a one-man project, is a community of contributors building backbone infrastructure for the entire mining industry.
```

Comment:
```suggestion
SRI 1.0.0 wouldn’t be possible without our contributors, supporters, working group, testers, and community.  Since our last update, our contributor base has grown even further!  What once started as a one-man project, is now a community of contributors building backbone infrastructure for the entire mining industry.
```